tests/test_europy.py

```diff
@@ -76,3 +89,22 @@ def test_minimum_functionality_example():
 @test(EXAMPLE_LABEL_NAME)
 def test_multiple_labels():
     return "Woah, what a fair unit test"
+
+
+@modeldetails('model_details.json') # this will override the current details in the report
```

Comment:
I am a bit confused about this, is this test just here to ship the model_details.json file into the report? I think that the file can live as code if that's the case. IMO code is usually the best documentation.

Reply:
I would be cool if we could just parse the docstring of a function decorated with @model_details and put that in the report, instead of having a whole test to go with it. Then a model_details decoration would just be a decoration that tells us that the comments/docstring here is meant to be included in the report.

Reply:
I was expecting this to be something you define outside of test, like in a training method. But we can do either. Problem with parsing a doc string is that we don't get a structure, like a JSON object or yaml. Have to rely on string parsing.